pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
 requires-python = ">=3.8"
 dependencies = [
   "aiohttp[speedups]~=3.8",
-  "gql[aiohttp,requests]~=3.4",
+  "gql[aiohttp,requests]>=3.5.1",
```

Comment:
Not sure if it was intentional, but this is dropping the major version constraint. I think you might want `>=3.5.1,<4` instead.

Reply:
Ah, yes, good point. (It was a purposeful bump of the minimum, but constraining the maximum is important still...)